<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="hit_button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stand_button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dealer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player_card1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_card3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_card4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_card5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>210</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer_card1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer_card2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer_card3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer_card4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Dealer_card5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>90</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
